doc/userguide.txt

```diff
@@ -1536,6 +1536,38 @@ You don't need to use these hooks- they are only here if you want to modify
 the behavior of uthash.  Hooks can be used to change how uthash allocates 
 memory, and to run code in response to certain internal events. 
 
+Preventing stadnard includes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For some targets, standard headers may not be available. To prevent uthash to
+pick up such headers, define `uthash_noheaders` before including uthash.h
+
+    #define uthash_noheaders
+    #include "uthash.h"
+
+Memory search and compare
```

Comment:
Actually, I think all this documentation should be folded into the following section on avoiding malloc/free. I can refactor the docs when I pull, though, so you don't actually have to worry about it.